pkg/cloudprovider/servicecontroller/servicecontroller.go

```diff
@@ -206,6 +215,7 @@ func (s *ServiceController) processDelta(delta *cache.Delta) (error, bool) {
 	case cache.Sync:
 		err, retry := s.createLoadBalancerIfNeeded(namespacedName, service, cachedService.service)
 		if err != nil {
+			s.eventRecorder.Event(service, "creating balancer failed", err.Error())
```

Comment:
Nit (because I believe these become part of our contract?): balancer -> loadbalancer ?